__mocks__/cairo/accountAbstraction/account_astraction.cairo

```diff
@@ -0,0 +1,348 @@
+            assert(twoFA0 == 0x04, 'Account tx: wrong 2FA-0');
+            let twoFA1 = *signature.at(3);
+            assert(twoFA1 == 0x05, 'Account tx: wrong 2FA-1');
+            let twoFA2 = *signature.at(4);
+            assert(twoFA2 == 0x06, 'Account tx: wrong 2FA-2');
+            let abstracted_hash = pedersen(
+                pedersen(pedersen(pedersen(pedersen(0, tx_hash), twoFA0), twoFA1), twoFA2),
+                4_felt252
+            ); // pedersen::pedersen () with Cairo v2.2.0
+            assert(
+                self._is_valid_signature(abstracted_hash, signature),
+                'Account tx: invalid signature'
+            );
+            starknet::VALIDATED
+        }
+
+        fn validate_deploy_account(self: @ContractState) -> felt252 {
+            let tx_info = get_tx_info().unbox();
+            let tx_hash = tx_info.transaction_hash;
+            // 1.print();
+            // tx_hash.print();
+
+            let signature = tx_info.signature;
+            assert(signature.len() == 5, 'Signature deplAcc needs 5 felts');
+            let twoFA0 = *signature.at(2);
+            assert(twoFA0 == 0x01, 'Account deplAcc: wrong 2FA-0');
+            let twoFA1 = *signature.at(3);
+            assert(twoFA1 == 0x02, 'Account deplAcc: wrong 2FA-1');
+            let twoFA2 = *signature.at(4);
+            assert(twoFA2 == 0x03, 'Account deplAcc: wrong 2FA-2');
+            let abstracted_hash = pedersen(
+                pedersen(pedersen(pedersen(pedersen(0, tx_hash), twoFA0), twoFA1), twoFA2),
+                4_felt252
+            ); // pedersen::pedersen () with Cairo v2.2.0
+            // 2.print();
+            // abstracted_hash.print();
+            // 3.print();
+            // let r = *signature.at(0);
+            // r.print();
+            // let s = *signature.at(1);
+            // s.print();
+            assert(
+                self._is_valid_signature(abstracted_hash, signature),
+                'Account depl: invalid signature'
+            );
+            starknet::VALIDATED
+        }
+
+        fn validate_declare(self: @ContractState) -> felt252 {
+            let tx_info = get_tx_info().unbox();
+            let tx_hash = tx_info.transaction_hash;
+            // 1.print();
+            // tx_hash.print();
+
+            let signature = tx_info.signature;
+            assert(signature.len() == 5, 'Signature decl needs 5 felts');
+            let twoFA0 = *signature.at(2);
+            assert(twoFA0 == 0x07, 'Declare : wrong 2FA-0');
+            let twoFA1 = *signature.at(3);
+            assert(twoFA1 == 0x08, 'Declare : wrong 2FA-1');
+            let twoFA2 = *signature.at(4);
+            assert(twoFA2 == 0x09, 'Declare : wrong 2FA-2');
+            let abstracted_hash = pedersen(
+                pedersen(pedersen(pedersen(pedersen(0, tx_hash), twoFA0), twoFA1), twoFA2),
+                4_felt252
+            ); // pedersen::pedersen () with Cairo v2.2.0
+            // 2.print();
```

Comment:
it's a mock contract, so maybe not that important but still we can consider removing commented code

Reply:
OK. Will be modified.